app/connectors/DepartureMovementP5Connector.scala

```diff
@@ -41,16 +39,16 @@ class DepartureMovementP5Connector @Inject() (config: AppConfig, http: HttpClien
     http.GET[DepartureMessages](serviceUrl)(implicitly, headers, ec)
   }
 
-  def getDepartureNotificationMessage[T <: Message](
+  def getMessage(
     departureId: String,
     messageId: String
-  )(implicit ec: ExecutionContext, hc: HeaderCarrier, reads: HttpReads[T]): Future[T] = {
+  )(implicit ec: ExecutionContext, hc: HeaderCarrier): Future[models.Message] = {
 
-    val headers = hc.withExtraHeaders(("Accept", "application/vnd.hmrc.2.0+json"))
+    val headers = hc.withExtraHeaders(("Accept", "application/vnd.hmrc.2.0+json-xml"))
 
     val serviceUrl = s"${config.commonTransitConventionTradersUrl}movements/departures/$departureId/messages/$messageId"
 
-    http.GET[T](serviceUrl)(reads, headers, ec)
+    http.GET[models.Message](serviceUrl)(implicitly, headers, ec)
```

Comment:
If you are only concerned about the body here you can use this endpoint (which just adds '/body' to the end of that url. Will save you having a read this into a model and you can just read it as a Node/NodeSeq:

https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/common-transit-convention-traders/2.0/oas/page#tag/Arrivals/operation/getMessageBodyRelatedToArrivalAndMessageID